### ISSUE TITLE
zippyshare doesn´t work

### ISSUE THREAD

**Issue by RavageVec** [pyload/pyload#1855]
I can´t get zippyshare downloads to work. I hope someone can help me.

INFO	HOSTER ZippyshareCom[75]: Updating file info...
260	23.09.2015 23:22:50	DEBUG	HOSTER ZippyshareCom[75]: File info: {'status': 2, 'url': u'http://www19.zippyshare.com/v/7fMeT2vU/file.html', 'size': 157286400.0, 'name': u'dou-eik1080p.part01.rar', 'pattern': {'S': u'150', 'U': u'MB', 'KEY': u'7fMeT2vU', 'N': u'dou-eik1080p.part01.rar'}}
261	23.09.2015 23:22:50	DEBUG	HOSTER ZippyshareCom[75]: Previous file info: {}
262	23.09.2015 23:22:50	INFO	HOSTER ZippyshareCom[75]: File name: dou-eik1080p.part01.rar
263	23.09.2015 23:22:50	INFO	HOSTER ZippyshareCom[75]: File size: 157286400 bytes
264	23.09.2015 23:22:50	INFO	HOSTER ZippyshareCom[75]: File status: online
265	23.09.2015 23:22:50	DEBUG	HOSTER ZippyshareCom[75]: LOAD URL http://www19.zippyshare.com/v/7fMeT2vU/file.html | cookies=[('zippyshare.com', 'ziplocale', 'en')] | get={} | req=None | decode=True | multipart=False | post={} | ref=False | just_header=False
266	23.09.2015 23:22:51	INFO	HOSTER ZippyshareCom[75]: Processing as free download...
267	23.09.2015 23:22:51	WARNING	CAPTCHA ZippyshareCom[75]: ReCaptcha | Key pattern not found
268	23.09.2015 23:22:53	INFO	HOSTER ZippyshareCom[75]: Downloading file...
269	23.09.2015 23:22:53	DEBUG	HOSTER ZippyshareCom[75]: DOWNLOAD URL /d/7fMeT2vU/693342/dou-eik1080p.part01.rar | cookies=True | get={} | disposition=True | post={} | ref=True
270	23.09.2015 23:22:53	DEBUG	Chunk 1 failed: (3, ' malformed')
271	23.09.2015 23:22:53	DEBUG	pycurl exception 3: malformed
272	23.09.2015 23:22:53	ERROR	pycurl error 3: malformed
273	23.09.2015 23:22:53	INFO	Debug Report written to debug_ZippyshareCom_23-09-2015_23-22-53.zip
274	23.09.2015 23:22:53	INFO	ADDON UnSkipOnFail: Looking for skipped duplicates of: dou-eik1080p.part01.rar (pid:3)
275	23.09.2015 23:22:53	INFO	ADDON UnSkipOnFail: No duplicates found
276	23.09.2015 23:22:53	DEBUG	All downloads processed 

TRACEBACK:
 Traceback (most recent call last):
  File "/usr/share/pyload/module/PluginThread.py", line 187, in run
    pyfile.plugin.preprocessing(self)
  File "/media/hdd/pyload/userplugins/internal/Hoster.py", line 184, in preprocessing
    return self._process(*args, **kwargs)
  File "/media/hdd/pyload/userplugins/internal/Hoster.py", line 179, in _process
    return self.process(self.pyfile)
  File "/media/hdd/pyload/userplugins/internal/SimpleHoster.py", line 288, in process
    self.download(self.link, disposition=self.DISPOSITION)
  File "/media/hdd/pyload/userplugins/internal/Hoster.py", line 434, in download
    progressNotify=self.pyfile.setProgress, disposition=disposition)
  File "/usr/share/pyload/module/network/Browser.py", line 88, in httpDownload
    name = self.dl.download(chunks, resume)
  File "/usr/share/pyload/module/network/HTTPDownload.py", line 129, in download
    self._download(chunks, resume)
  File "/usr/share/pyload/module/network/HTTPDownload.py", line 260, in _download
    raise ex
error: (3, '<url> malformed')
 

FRAMESTACK:

Frame preprocessing in /media/hdd/pyload/userplugins/internal/Hoster.py at line 184
	                self = <Hoster ZippyshareCom>
	                args = (<DownloadThread(Thread-2, started daemon 1972638832)>,)
	              kwargs = {}

Frame _process in /media/hdd/pyload/userplugins/internal/Hoster.py at line 179
	                self = <Hoster ZippyshareCom>
	              thread = <DownloadThread(Thread-2, started daemon 1972638832)>

Frame process in /media/hdd/pyload/userplugins/internal/SimpleHoster.py at line 298
	              pyfile = PyFile 75: dou-eik1080p.part01.rar@ZippyshareCom
	                self = <Hoster ZippyshareCom>

Frame download in /media/hdd/pyload/userplugins/internal/Hoster.py at line 436
	             cookies = True
	                 key = 'ref'
	                 val = True
	                 get = {}
	                 url = '/d/7fMeT2vU/693342/dou-eik1080p.part01.rar'
	                self = <Hoster ZippyshareCom>
	   download_location = u'/media/hdd/downloads/Es.ist.kompliziert.2015.German.DL.1080p.BluRay.x264-DOUCEMENT - filecrypt.cc'
	            filename = u'/media/hdd/downloads/Es.ist.kompliziert.2015.German.DL.1080p.BluRay.x264-DOUCEMENT - filecrypt.cc/dou-eik1080p.part01.rar'
	     download_folder = u'/media/hdd/downloads/'
	            location = u'/media/hdd/downloads/Es.ist.kompliziert.2015.German.DL.1080p.BluRay.x264-DOUCEMENT - filecrypt.cc'
	         disposition = True
	                post = {}
	                 ref = True

Frame httpDownload in /usr/share/pyload/module/network/Browser.py at line 88
	             cookies = True
	      progressNotify = <bound method PyFile.setProgress of PyFile 75: dou-eik1080p.part01.rar@ZippyshareCom>
	              resume = True
	                 url = '/d/7fMeT2vU/693342/dou-eik1080p.part01.rar'
	                self = <module.network.Browser.Browser object at 0x288cab0>
	                 get = {}
	            filename = u'/media/hdd/downloads/Es.ist.kompliziert.2015.German.DL.1080p.BluRay.x264-DOUCEMENT - filecrypt.cc/dou-eik1080p.part01.rar'
	         disposition = True
	              chunks = 3
	                post = {}
	                 ref = True

Frame download in /usr/share/pyload/module/network/HTTPDownload.py at line 145
	              chunks = 3
	                self = <module.network.HTTPDownload.HTTPDownload instance at 0x290e0a8>
	                code = 3
	                   e = error(3, '<url> malformed')
	              resume = False

Frame _download in /usr/share/pyload/module/network/HTTPDownload.py at line 260
	               chunk = <HTTPChunk id=0, size=-1, arrived=0>
	                done = False
	                  ex = error(3, '<url> malformed')
	              chunks = 3
	                curl = <pycurl.Curl object at 0x29beba8>
	       chunksCreated = False
	               errno = 3
	                self = <module.network.HTTPDownload.HTTPDownload instance at 0x290e0a8>
	                 ret = 0
	              failed = [<HTTPChunk id=0, size=-1, arrived=0>]
	                init = <HTTPChunk id=0, size=-1, arrived=0>
	         num_handles = 0
	                 msg = '<url> malformed'
	          chunksDone = set()
	     lastFinishCheck = 0
	              resume = False
	            err_list = [(<pycurl.Curl object at 0x29beba8>, 3, '<url> malformed')]
	             ok_list = []
	               num_q = 0
	                   c = (<pycurl.Curl object at 0x29beba8>, 3, '<url> malformed')
	                   t = 1443050573.798284
	       lastTimeCheck = 0

PLUGIN OBJECT DUMP: 

	          CHECK_FILE = True
	       CHECK_TRAFFIC = False
	             COOKIES = [('zippyshare.com', 'ziplocale', 'en')]
	         DIRECT_LINK = None
	         DISPOSITION = True
	         FILE_ERRORS = [('Html error',
  '\\A(?:\\s*<.+>)?((?:[\\w\\s]*(?:[Ee]rror|ERROR)\\s*\\:?)?\\s*\\d{3})(?:\\Z|\\s+)'),
 ('Request error', '([Aa]n error occured while processing your request)'),
 ('Html file', '\\A\\s*<!DOCTYPE html')]
	        LEECH_HOSTER = False
	        LINK_PATTERN = None
	LINK_PREMIUM_PATTERN = "document.location = '(.+?)'"
	       LOGIN_ACCOUNT = False
	       LOGIN_PREMIUM = False
	        NAME_PATTERN = '(<title>Zippyshare.com - |"/)(?P<N>[^/]+)(</title>|";)'
	   NAME_REPLACEMENTS = []
	     OFFLINE_PATTERN = 'does not exist (anymore )?on this server<'
	        SIZE_PATTERN = '>Size:.+?">(?P<S>[\\d.,]+) (?P<U>[\\w^_]+)'
	   SIZE_REPLACEMENTS = []
	       TEXT_ENCODING = True
	    URL_REPLACEMENTS = []
	             account = None
	             captcha = <Captcha Captcha>
	         chunk_limit = -1
	           direct_dl = False
	                html = u'<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">\n    <html xmlns:fb="http://ogp.me/ns/fb#">\n        <head>\n            <title>Zippyshare.com</title>\n                <meta http-equiv="content-type" content="text/html; charset=utf-8">\n                <meta name="robots" content="index, follow, noarchive, all">\n                <meta name="keywords" content="">\n                <meta name="description" content="">\n            <meta property="og:title" content="Private file" />\n            <meta property="og:url" content="http://www19.zippyshare.com/v/7fMeT2vU/file.html" />\n            <meta property="fb:app_id" content="186641184698333"/>\n            <meta property="og:description" content="Zippyshare.com - Free File Hosting" />\n            <link rel="shortcut icon" href="/images/favicon.ico">\n            <link rel="stylesheet" href="/css/main.css" type="text/css">\n            <link rel="stylesheet" href="/css/embed.css" type="text/css" media="screen">\n            <link rel="stylesheet" href="/css/colorpicker.css" type="text/css" media="screen">\n            <link rel="stylesheet" href="/css/layout.css" type="text/css" media="screen">\n            <link href="/css/jplayer.css" rel="stylesheet" type="text/css">\n            \n            <script type="text/javascript" src="/js/jquery-1.11.2.min.js"></script>\n            <script type="text/javascript" src="/js/jquery.blockUI.2.7.js"></script>\n            <script type="text/javascript" src="/js/jquery.corner.js"></script>\n            <script type="text/javascript" src="/js/jquery.simplemodal.embed.js"></script>\n            <script type="text/javascript" src="/js/embed.js"></script>\n            <script type="text/javascript" src="/js/swfobject.js"></script>\n            <script type="text/javascript" src="/js/colorpicker.js"></script>\n            <script type=\'text/javascript\' src=\'/js/jquery.cookie.js\'></script>\n            <script type=\'text/javascript\' src=\'/js/json2.js\'></script>\n            <script src=\'https://www.google.com/recaptcha/api.js?render=explicit\'></script>\n            <script type="text/javascript" src="/js/jquery.jplayer.min.js"></script>\n            <script type="text/javascript" src="/js/jquery.qrcode-0.11.0.min.js"></script>\n            <script type="text/javascript">\n                if (top != self)\n                    if (location)\n                        top.location.replace(self.location.href);\n                    else\n                        top.document.location.replace(self.document.location.href);\n                $(function () {\n                    $(\'div.inner_menu\').each(function () {\n                        var t = $(this).corner("round 4px").parent().css(\'padding\', \'2px\').corner("round 6px");\n                        eval(t);\n                    });\n                    $(\'div.inner_main\').each(function () {\n                        var t = $(this).corner("round 4px").parent().css(\'padding\', \'2px\').corner("round 6px");\n                        eval(t);\n                    });\n                });\n            </script>\n            <script type="text/javascript">\n                $(document).ready(function () {\n                    $("#lang-one li").hover(\n                            function () {\n                                $("ul", this).fadeIn("fast");\n                            },\n                            function () {\n                            }\n                    );\n                    if (document.all) {\n                        $("#lang-one li").hoverClass("sfHover");\n                    }\n                });\n\n                $.fn.hoverClass = function (c) {\n                    return this.each(function () {\n                        $(this).hover(\n                                function () {\n                                    $(this).addClass(c);\n                                },\n                                function () {\n                                    $(this).removeClass(c);\n                                }\n                        );\n                    });\n                };\n\n                addFileToMyFiles = function (skey) {\n                    $.ajax({\n                        url: "/rest/file/addFileToMyFiles",\n                        type: "GET",\n                        data: {key: skey},\n                        dataType: "json",\n                        success: function (data) {\n                            if (data == 0) {\n                                alert(\'File has been added to your File Manager\');\n                            } else {\n                                alert(\'An error has occured\');\n                            }\n                        },\n                        error: function (date) {\n                            alert(\'An error has occured\');\n                        }\n                    });\n                };\n\n                setLocale = function (locale) {\n                    var date = new Date();\n                    date.setTime(date.getTime() + 60 * 60 * 24 * 365 * 10);\n                    $.cookie("ziplocale", locale, {domain: ".zippyshare.com", expires: date});\n                    location.reload(true);\n                };\n            </script>\n            <script type="text/javascript">\n\n                var _gaq = _gaq || [];\n                _gaq.push([\'_setAccount\', \'UA-28004937-1\']);\n                _gaq.push([\'_setDomainName\', \'zippyshare.com\']);\n                _gaq.push([\'_trackPageview\']);\n\n                (function () {\n                    var ga = document.createElement(\'script\');\n                    ga.type = \'text/javascript\';\n                    ga.async = true;\n                    ga.src = (\'https:\' == document.location.protocol ? \'https://ssl\' : \'http://www\') + \'.google-analytics.com/ga.js\';\n                    var s = document.getElementsByTagName(\'script\')[0];\n                    s.parentNode.insertBefore(ga, s);\n                })();\n\n            </script>\n        </head>\n        <body>\n            <div class="container">\n\n                <div class="login">\n                    <form method="post" action="http://www.zippyshare.com/services/login" name="login">\n                            <span><img src="/images/icons/user.png" align="absmiddle" height="14px" style="padding-right: 2px;" />Username:</span>\n                            <input name="login" type="text" size="22" style="height:11px; margin-right: 4px; font-size: 9px; color: #15598D;" />\n                            <span><img src="/images/icons/key.png" align="absmiddle" height="14px" style="padding-right: 2px;" />Password:</span>\n                            <input name="pass" type="password" size="22" style="height:11px; font-size: 9px; color: #15598D;" />\n                            <input type="submit" class="przycisk" value="Login" style="height:17px; width: 60px; vertical-align: bottom;" /><br />\n                            <div style="float: right; margin-top: 4px;">\n                                <input type="checkbox" name="remember">\n                                <font style="color: #777; font-size: 9px; vertical-align: text-top;">Remember Me |</font>\n                                <a style="color: #777; text-decoration: none; font-size: 9px; vertical-align: text-top;" href="http://www.zippyshare.com/sites/forgot.jsp">Forgot Password?</a>\n                            </div>\n                        </form>\n                    </div>\n\n                <div style="text-align: center; margin: 0 auto;">\n                    <a href="http://www.zippyshare.com/" target="_self"><img src="/images/logo.png" alt="ZippyShare.com - Free File Hosting" style="border: 0pt; margin-left: 0px; margin-bottom: 10px;" /></a><br />\n                </div>\n                <div class="outer_menu">\n                    <div class="inner_menu">\n                        <span style="float: left;">\n                            <a href="http://www.zippyshare.com" class="menu">Upload Files</a> |\n                            <a href="#" class="menu">F.A.Q</a> |\n                            <a href="http://www.zippyshare.com/sites/uploader.jsp" class="menu">Upload Tools</a> |\n                            <a href="http://www.blog.zippyshare.com/" class="menu">Our Blog</a> |\n                            <a href="http://www.support.zippyshare.com/index.php?act=tickets&code=open" target="_blank" class="menu">Contact Us</a>\n                        </span>\n                        <span style="float: right; margin-top: -2px;">\n                            <ul id="lang-one" class="lang">\n                                <li>\n                                    <a href="/view.jsp?locale=en&key=7fMeT2vU" class="flagen" title="English"><img src="/images/arrow_langs.gif" alt="" style="float: left; margin: 8px 5px 0px -5px;" />English</a>\n                                    <ul>\n                                        <li><a href="#" onclick="setLocale(\'de\');" class="flagde" title="Deutsch">Deutsch</a></li>\n                                            <li><a href="#" onclick="setLocale(\'nl\');" class="flagnl" title="Nederlands">Nederlands</a></li>\n                                            <li><a href="#" onclick="setLocale(\'fr\');" class="flagfr" title="Francais">Francais</a></li>\n                                            <li><a href="#" onclick="setLocale(\'hu\');" class="flaghu" title="Magyar">Magyar</a></li>\n                                            <li><a href="#" onclick="setLocale(\'lt\');" class="flaglt" title="Lietuvi\u0173">Lietuvi\u0173</a></li>\n                                            <li><a href="#" onclick="setLocale(\'pl\');" class="flagpl" title="Polski">Polski</a></li>\n                                            <li><a href="#" onclick="setLocale(\'pt\');" class="flagpt" title="Portugu\xeas">Portugu\xeas</a></li>\n                                            <li><a href="#" onclick="setLocale(\'ro\');" class="flagro" title="Rom\xe2n\u0103">Rom\xe2n\u0103</a></li>\n                                            <li><a href="#" onclick="setLocale(\'ru\');" class="flagru" title="\u0420\u0443\u0441\u0441\u043a\u0438\u0439">\u0420\u0443\u0441\u0441\u043a\u0438\u0439</a></li>\n                                            <li><a href="#" onclick="setLocale(\'es\');" class="flages" title="Espa\xf1ol">Espa\xf1ol</a></li>\n                                            <li><a href="#" onclick="setLocale(\'sv\');" class="flagse" title="Svenska">Svenska</a></li>\n                                            <li><a href="#" onclick="setLocale(\'tr\');" class="flagetr" title="T\xfcrk\xe7e">T\xfcrk\xe7e</a></li>\n                                            </ul>\n                                </li>\n                            </ul>\n                        </span>\n                        <span style="float: right; margin-top: 1px; font-size: 13px; font-weight: bolder; color: black;">Select Language:\xa0\xa0</span>\n                    </div>\n                </div>\n                <div class="outer_main">\n                    <div class="inner_main">\n                        <table cellspacing="0" cellpadding="0" width="984px;" style="background: transparent url(/images/folder_logo.jpg); background-repeat: no-repeat; background-position: right top;">\n                            <tr>\n                                <td>\n                                    <div id="lrbox">\n                                        \n\n\n\n\n\n\n\n\n\n\n\n\n    <div class="left" style="width: 700px;">\n        <font style="line-height:20px ;font-size: 16px; color: #CC0000; font-weight: bolder;">You have requested the file:</font><br />\n        <font style="line-height:20px; font-size: 14px; font-weight: bolder;">Name:</font>          <font style="line-height:20px; font-size: 14px;"><img style="margin-left: 10px;" src="/fileName?key=7fMeT2vU&size=small" alt="file name"></font><br />\n        <font style="line-height:18px; font-size: 13px; font-weight: bold;">Size:</font>            <font style="line-height:18px; font-size: 13px;">150 MB</font><br />\n        <font style="line-height:18px; font-size: 13px; font-weight: bold;">Uploaded:</font>        <font style="line-height:18px; font-size: 13px;">23-09-2015 17:59</font><br />\n        <font style="line-height:18px; font-size: 13px; font-weight: bold;">Last download:</font>        <font style="line-height:18px; font-size: 13px;">24-09-2015 01:19</font><br />\n        <div style="width: 700px; height: 25px; margin-top: 4px; margin-left: -2px;">\n            <div class="addthis_sharing_toolbox"></div>\n            <script type="text/javascript" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=ra-4d7009770839a69f" async="async"></script>\n        </div>\n    </div>\n    <div class="right" style="margin-top: 15px; width: 251px;">\n        \n            \n            \n                <div style="margin-left: -22px; margin-top: -5px; text-align: center;width: 303px;">\n                    <a id="dlbutton"  href="#">\n                        <img src="/images/download.png" alt="Download" border="0">\n                    </a>\n                    \n                    \n\n\n\n<script type="text/javascript">\n    var a = 520007;\n    var b = 742589;\n    document.getElementById(\'dlbutton\').omg = "f";\n    if (document.getElementById(\'dlbutton\').omg != \'f\') {\n       a = Math.ceil(a/3);\n    } else {\n       a = Math.floor(a/3);\n    }\n    document.getElementById(\'dlbutton\').href = "/d/7fMeT2vU/"+(a + 520007%b)+"/dou-eik1080p.part01.rar";\n    if (document.getElementById(\'fimage\')) {\n        document.getElementById(\'fimage\').href = "/i/7fMeT2vU/"+(a + 520007%b)+"/dou-eik1080p.part01.rar";\n    }\n</script>\n                </div>\n            \n        \n    </div>\n    <div class="center" style="text-align: center;">\n        <font style="font-size: 10px; letter-spacing: 3px; word-spacing: 2px;">Advertisement</font>\n    </div>\n    <div class="center_ad">\n        <div style="float: left; width: 328px; height: 276px; background-image: url(/images/zs_small_tab.gif);">\n            <div style="margin-left: 14px; margin-top: 14px; height: 300px; width: 300px;">\n                <iframe id="a0e758de" name="a0e758de" src="//adsvidsdouble.com/afr.php?refresh=N&zoneid=188403" frameborder="0" scrolling="no" width="300" height="250"><a href="//adsvidsdouble.com/ck.php?n=a706eda3" target="_blank"><img src="//adsvidsdouble.com/avw.php?zoneid=188403&n=a706eda3" border="0" alt="" /></a></iframe> \n                 \n                \n            </div>\n        </div>\n        <div style="float: left; width: 328px; height: 276px; background-image: url(/images/zs_small_tab.gif);">\n            <div style="margin-left: 14px; margin-top: 14px; height: 300px; width: 300px;">\n               <iframe src="//go.padstm.com/?id=189775&t=iframe" style="width:300px;height:250px;border:0;overflow:hidden;"></iframe>\n                \n            </div>\n        </div>\n        <div style="float: left; width: 328px; height: 276px; background-image: url(/images/zs_small_tab.gif);">\n            <div style="margin-left: 14px; margin-top: 14px; height: 300px; width: 300px;">\n                <iframe src="//go.padstm.com/?id=189777&t=iframe" style="width:300px;height:250px;border:0;overflow:hidden;"></iframe>\n                 \n            </div>\n        </div>\n    </div>\n\n<div class="slickbox">\n                                            <div class="center">\n                                                <div class="ramka_container" style="max-width: 1185px;padding:0 0 5px 0; margin-top:20px;">\n    <div class="belka" style="text-align: center;">\n        <img style="vertical-align: absmiddle;" src="/images/icons/newspaper.png"/>\n        <font style="font-size: 13px; font-weight: bolder; text-decoration: underline;">Zippyshare.com News:</font>\n    </div>\n    <div class="pojemnik">\n        <div>\n                <div>\n                    <img style="float: left;" align="middle" src="/images/icons/new.png" alt="new"/>\n                    <div style="float: left; margin-left: 5px; width:600px;">\n                        <strong><a style="font-size:12px; line-height: 18px;" href="http://blog.zippyshare.com/?p=754">... and the first update of 2015 is behind us :-)</a></strong>\n                    </div>\n                    <br/>\n                    <div style="margin-left: 20px; font-size: 10px;">11 Jan 2015 18:02</div>\n                </div>\n                <div style="margin: 1px 0pt 0pt 20px; font-size:12px; line-height: 18px;">\n                    After quite a long time we finally managed to put together o meaningful update.\r\n\r\n<strong><u>What has been changed?</u></strong>\r\n\r\n<strong>- Link format.</strong> Links are now alphanumeric and case...\n                </div>\n                <br/>\n            </div>\n        <div>\n                <div>\n                    <img style="float: left;" align="middle" src="/images/icons/new.png" alt="new"/>\n                    <div style="float: left; margin-left: 5px; width:600px;">\n                        <strong><a style="font-size:12px; line-height: 18px;" href="http://blog.zippyshare.com/?p=686">Maintenance / Technical entry</a></strong>\n                    </div>\n                    <br/>\n                    <div style="margin-left: 20px; font-size: 10px;">21 Dec 2013 15:04</div>\n                </div>\n                <div style="margin: 1px 0pt 0pt 20px; font-size:12px; line-height: 18px;">\n                    Hey folks,\r\n\r\nThis is maintenance/technical entry, we will update it when something goes wrong.\r\n\r\n<strong>Edit 21.12.2013:</strong>\r\nSorry for a little slowdowns on 41/42 and 43/44, we had to swap a ...\n                </div>\n                <br/>\n            </div>\n        <div>\n                <div>\n                    <img style="float: left;" align="middle" src="/images/icons/new.png" alt="new"/>\n                    <div style="float: left; margin-left: 5px; width:600px;">\n                        <strong><a style="font-size:12px; line-height: 18px;" href="http://blog.zippyshare.com/?p=662">Quick update</a></strong>\n                    </div>\n                    <br/>\n                    <div style="margin-left: 20px; font-size: 10px;">26 Oct 2013 17:51</div>\n                </div>\n                <div style="margin: 1px 0pt 0pt 20px; font-size:12px; line-height: 18px;">\n                    We are still alive!\r\n\r\nSome of You were concerned by the lack of new messages on our blog so we took the opportunity to give You an update. Everything is going fine. We don\'t have any particularly exc...\n                </div>\n                <br/>\n            </div>\n        </div>\n</div></div>\n                                            <div class="left" style="padding-top: 10px; text-align: center;">\n                                                <font class="bold">To upload a file just follow these simple steps:</font>\n                                            </div>\n                                            <div class="right" style="padding-top: 10px; text-align: center;">\n                                                <font class="bold">Benefits of using Zippyshare:</font>\n                                            </div>\n                                            <div class="left">\n                                                <b>1)</b> Select a file to send by clicking the "Browse" button. You can then select photos, audio, video, documents or anything else you want to send. The maximum file size is 200 MB.<br><br><b>2)</b> Click the "Start Upload" button to start uploading the file. You will see the progress of the file transfer. Please don\'t close your browser window while uploading or it will cancel the upload.<br><br><b>3)</b> After a succesfull upload you\'ll receive a unique link to the download site, which you can place anywhere: on your homepage, blog, forum or send it via IM or e-mail to your friends.<br><br></div>\n                                                <div class="right" style="margin-top: -3px;">\n                                                    <font class="list">\n                                                    <img src="/images/icons/tick.png" alt="">Zippyshare.com is completely free, reliable and popular way to store \xa0\xa0\xa0\xa0\xa0 files online.<br>\n                                                <img src="/images/icons/tick.png" alt="">We offer fast download speeds.<br>\n                                                <img src="/images/icons/tick.png" alt="">The maximum filesize for a single file is 200 MB.<br>\n                                                <img src="/images/icons/tick.png" alt="">The file can be downloaded at any time and as often as you need it.<br>\n                                                <img src="/images/icons/tick.png" alt="">File Life: 30 days after no activity.<br>\n                                                <img src="/images/icons/tick.png" alt="">No ridiculous queues!<br>\n                                                <img src="/images/icons/tick.png" alt="">No limits!<br>\n                                                </font>\n                                            </div>\n                                        </div>\n                                        <div class="center" style="text-align: center; margin-bottom: 10px; margin-top: 15px;">\n                                            <font style="font-size: 10px; color: #000000; font-weight: bolder;">Report illegal files, please\xa0<a href="http://www.support.zippyshare.com/index.php?act=tickets&code=open" class="abuse">click here </a>and send full link to us!</font>\n                                        </div>\n                                    </div>\n                                </td>\n                            </tr>\n                        </table>\n                    </div>\n                </div>\n                <div id="bottom">\n                    <font style="font-size: 12px; color: #000000; font-weight: bolder;">\xa9 2006-2015 Zippyshare.com. All rights reserved.</font><br />\n                    <a target="_blank" href="../../terms_and_conditions_of_use.html" class="PPToS">Terms and Conditions</a> <font style="font-size: 10px; color: #000000;">|</font> <a target="_blank" href="../../dmca.html" class="PPToS">DMCA Policy</a>\n                    <script type="text/javascript" src="//go.onclasrv.com/apu.php?zoneid=180376"></script>\n                            </div>\n            </div>\n            <div id="qrCode">\n\n            </div>\n        </body>\n        <script>\n            $(function () {\n                var cookiesEnabled = function () {\n                    if (navigator.cookieEnabled)\n                        return true;\n\n                    // set and read cookie\n                    document.cookie = "cookietest=1";\n                    var ret = document.cookie.indexOf("cookietest=") != -1;\n                    // delete cookie\n                    document.cookie = "cookietest=1; expires=Thu, 01-Jan-1970 00:00:01 GMT";\n                    return ret;\n                }\n\n                if (typeof displayReCaptcha !== \'undefined\') {\n                    var tryLoad = function () {\n                        if (typeof (grecaptcha) !== \'undefined\' && grecaptcha !== null) {\n                            $(\'.rc-anchor-standard\').css(\'background-color\', \'transparent\');\n                            displayReCaptcha();\n                        } else {\n                            setTimeout(tryLoad, 100);\n                        }\n                    }\n\n                    tryLoad();\n                }\n\n                if (!cookiesEnabled()) {\n                    $.blockUI({\n                        message: \'Cookies must be enabled to download\',\n                        css: {\n                            border: \'none\',\n                            padding: \'15px\',\n                            backgroundColor: \'#FF0000\',\n                            \'-webkit-border-radius\': \'10px\',\n                            \'-moz-border-radius\': \'10px\',\n                            opacity: .5,\n                            color: \'#fff\'\n                        }\n                    });\n                    $(\'.blockOverlay\').attr(\'title\', \'Click to unblock\').click($.unblockUI);\n                }\n\n                var a = navigator.userAgent || navigator.vendor || window.opera;\n                if (/android|android.+mobile|avantgo|bada\\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\\/|plucker|pocket|psp|symbian|treo|up\\.(browser|link)|vodafone|wap|windows (ce|phone)|xda|xiino/i.test(a) || /1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\\-(n|u)|c55\\/|capi|ccwa|cdm\\-|cell|chtm|cldc|cmd\\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\\-s|devi|dica|dmob|do(c|p)o|ds(12|\\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\\-|_)|g1 u|g560|gene|gf\\-5|g\\-mo|go(\\.w|od)|gr(ad|un)|haie|hcit|hd\\-(m|p|t)|hei\\-|hi(pt|ta)|hp( i|ip)|hs\\-c|ht(c(\\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\\-(20|go|ma)|i230|iac( |\\-|\\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\\/)|klon|kpt |kwc\\-|kyo(c|k)|le(no|xi)|lg( g|\\/(k|l|u)|50|54|e\\-|e\\/|\\-[a-w])|libw|lynx|m1\\-w|m3ga|m50\\/|ma(te|ui|xo)|mc(01|21|ca)|m\\-cr|me(di|rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\\-2|po(ck|rt|se)|prox|psio|pt\\-g|qa\\-a|qc(07|12|21|32|60|\\-[2-7]|i\\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\\-|oo|p\\-)|sdk\\/|se(c(\\-|0|1)|47|mc|nd|ri)|sgh\\-|shar|sie(\\-|m)|sk\\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\\-|v\\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\\-|tdg\\-|tel(i|m)|tim\\-|t\\-mo|to(pl|sh)|ts(70|m\\-|m3|m5)|tx\\-9|up(\\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|xda(\\-|2|g)|yas\\-|your|zeto|zte\\-/i.test(a.substr(0, 4))) {\n                } else {\n                    $("#qrCode").qrcode({\n                        "size": 100,\n                        "color": "#3a3",\n                        "text": "http://www19.zippyshare.com/v/7fMeT2vU/file.html"\n                    });\n                }\n            });\n        </script>\n    </html>\n'
	                info = {'folder': u'dou-eik1080p.part01.rar',
 'name': u'dou-eik1080p.part01.rar',
 'pattern': {'KEY': u'7fMeT2vU',
             'N': u'dou-eik1080p.part01.rar',
             'S': u'150',
             'U': u'MB'},
 'size': 157286400.0,
 'status': 2,
 'url': u'http://www19.zippyshare.com/v/7fMeT2vU/file.html'}
	                  js = <module.common.JsEngine.JsEngine instance at 0x25ee468>
	          last_check = None
	       last_download = ''
	            leech_dl = False
	             limitDL = 0
	                link = '/d/7fMeT2vU/693342/dou-eik1080p.part01.rar'
	             multiDL = True
	             premium = False
	              pyfile = PyFile 75: dou-eik1080p.part01.rar@ZippyshareCom
	              pyload = <__main__.Core object at 0x7625cf30>
	                 req = <module.network.Browser.Browser object at 0x288cab0>
	     resume_download = True
	             retries = {}
	          retry_free = False
	              thread = <DownloadThread(Thread-2, started daemon 1972638832)>
	          wait_until = 0
	             waiting = False
	       wantReconnect = False

PYFILE OBJECT DUMP: 

	               _size = 0
	               abort = False
	              active = False
	               error = ''
	                  id = 75
	                lock = <_RLock owner=None count=0>
	                   m = <module.database.FileDatabase.FileHandler instance at 0x7626cee0>
	         maxprogress = 100
	                name = u'dou-eik1080p.part01.rar'
	               order = 0
	           packageid = 3
	              plugin = <Hoster ZippyshareCom>
	         pluginclass = <class 'userplugins.hoster.ZippyshareCom.ZippyshareCom'>
	        pluginmodule = <module 'userplugins.hoster.ZippyshareCom' from '/media/hdd/pyload/userplugins/hoster/ZippyshareCom.pyc'>
	          pluginname = u'ZippyshareCom'
	            progress = 0
	         reconnected = False
	                size = 0
	              status = 8
	          statusname = None
	                 url = u'http://www19.zippyshare.com/v/7fMeT2vU/file.html'
	           waitUntil = 0

CONFIG: 

{'desc': 'ZippyshareCom',
 'outline': 'Zippyshare.com hoster plugin',
 'use_premium': {'desc': 'Use premium account if available',
                 'type': 'bool',
                 'value': True}}
